composer.json

```diff
@@ -1,31 +1,57 @@
 {
     "name": "terminal42/contao-inserttags",
-    "description": "Contao extension to create InsertTags to replace content in front- or backend.",
+    "description": "Contao extension to create custom insert tags.",
     "keywords": ["contao", "inserttags", "backend", "frontend"],
-    "type": "contao-module",
-    "license": "LGPL-3.0+",
+    "type": "contao-bundle",
+    "license": "MIT",
     "authors": [
         {
             "name": "terminal42 gmbh",
             "homepage": "https://www.terminal42.ch"
         }
     ],
+    "funding": [
+        {
+            "type": "github",
+            "url": "https://github.com/terminal42"
+        },
+        {
+            "type": "other",
+            "url": "https://ko-fi.com/terminal42"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/terminal42/contao-inserttags/issues",
+        "source": "https://github.com/terminal42/contao-inserttags"
+    },
     "require": {
-        "php": "^5.3.2 || ^7.0",
-        "contao/core-bundle": "^3.5.1 || ^4.4",
-        "contao-community-alliance/composer-plugin": "~2.4 || ~3.0"
+        "php": "^7.4 || ^8.0",
```

Comment:
Imho, we can easily go for `^8.0` only. It's v2 after all. Then you could also use constructor property promotion and save some code :)